packages/core/security/core-security-server-internal/src/utils/default_implementation.test.ts

```diff
@@ -22,4 +22,19 @@ describe('getDefaultSecurityImplementation', () => {
       expect(user).toBeNull();
     });
   });
+
+  describe('audit.asScoped', () => {
+    it('returns null', async () => {
+      const logger = implementation.audit.asScoped({} as any);
+      expect(logger.log({ message: 'something' })).toBeNull();
```

Comment:
__question:__ hmm, why do we return\expect `null` when the type definition claims that it should be `void`?

Reply:
Great catch! I've updated it.